lib/LANraragi/Utils/Database.pm

```diff
@@ -135,21 +136,23 @@ sub add_pagecount ( $redis, $id ) {
     $redis->hset( $id, "pagecount", scalar @images );
 }
 
+# Retrieves the archive's info as hash (empty if not found)
+sub get_archive ($id) {
+    my $redis = LANraragi::Model::Config->get_redis;
+    my %hash  = h_get_archive( $redis, $id );
+    $redis->quit();
+    return %hash;
+}
+
+sub h_get_archive ( $redis, $id ) {
```

Comment:
Here I wanted to name the call `$redis->hgetall($id);` to make it immediately understandable and searchable (during a refactoring). It can also be used transactionally, like in `get_archive_json_multi()` and it restricts the call to `hgetall` inside the Database package. I think it might help in building tests.

Anyway being the only method changed like this, is not really helpful and I wanted to read your opinion before replacing its other calls around.

Reply:
I'm not really a fan, sorry!  
I don't see either how useful it'd be for testing compared to just mocking hgetall directly like we do now. 🤔

Reply:
It's fine, I shouldn't have left it in from the beginning.